src/core/parsers/nmea/GPSFix.cpp

```diff
@@ -234,10 +234,7 @@ GPSFix::GPSFix()
     visibleSatellites = 0;
 }
 
-GPSFix::~GPSFix()
-{
-    // TODO Auto-generated destructor stub
-}
+GPSFix::~GPSFix() {}
```

Comment:
Can we
```suggestion
```
and also remove the  [`virtual ~GPSFix();`](https://github.com/Team-Abhiyaan/mosaic_gnss_driver/blob/1c96b25fe65db8a2c3cd5187a61cfbec8266c586/include/mosaic_gnss_driver/parsers/nmeaparse/GPSFix.h#L121) from `GPSFix.h`

From what I know the compiler auto generates such destructors.

Reply:
Makes sense. I suppose we won't be using `GPSFix` or `GPSService` as base classes for any other classes. If we are going to inherit from them, then we'll need to keep the virtual function declarations. Anyways, I am removing them for now.